VideoLocker/res/layout/discussion_responses_thread_row.xml

```diff
@@ -37,6 +37,15 @@
             android:textSize="@dimen/edx_small"
             tools:text="Thread body" />
 
+        <TextView
+            android:id="@+id/discussion_responses_thread_row_visibility_text_view"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="@dimen/discussion_responses_standard_margin"
+            android:textColor="@color/edx_grayscale_neutral_base"
+            android:textSize="@dimen/edx_xxx_small"
```

Comment:
> Maybe the discussion_responses_small_text style can be applied here? That style should probably be unbundled as well (maybe inheriting from regular_edx_gray_base_text), so that it can also be applied to the other text views.

Good catch. I'll move that style to `text_styles.xml` file as well.